libs/build_dod_dataset.py

```diff
@@ -105,6 +112,120 @@ def get_projections_df():
     "Unassigned": NULL_VALUE
 }
 
+def get_state_and_counties():
+    # go to the JHUDataset to get this, for w/e reason
+    timeseries = JHUDataset.local().timeseries()
```

Comment:
If you use the `FIPSPopulation` dataset source, it has more "ground truth" county data - don't think all of the counties are in the JHU dataset.

Also, unrelated, over the week i've been thinking about ways to make the interface for datasets better - even though I wrote it, i've really come to hate the `.local().timeseries()`.  
Do you have an initial reaction to calling `TimeseriesDataset.load_jhu_dataset()` to get the data? Willingly taking any and all suggestions

Reply:
oh perfect i had been looking for this